osu.Game.Rulesets.Diva/Scoring/DivaHitWindows.cs

```diff
@@ -5,16 +5,17 @@ namespace osu.Game.Rulesets.Diva.Scoring
 {
     public class DivaHitWindows : HitWindows
     {
-        public override bool IsHitResultAllowed(HitResult result) {
-            switch (result) 
+        public override bool IsHitResultAllowed(HitResult result)
+        {
+            switch (result)
             {
-            case HitResult.Perfect:
-            case HitResult.Great:
-            case HitResult.Good:
-            case HitResult.Ok:
-            case HitResult.Meh:
-            case HitResult.Miss:
-                return true;
+                case HitResult.Perfect:
+                case HitResult.Great:
+                case HitResult.Good:
+                case HitResult.Ok:
+                case HitResult.Meh:
+                case HitResult.Miss:
+                    return true;
```

Comment:
Again, personal preference when it comes to switch block indent, I personally disagree with this particular style